packages/shared/lib/contexts/governance/utils/sortProposals.ts

```diff
@@ -21,16 +22,42 @@ export function sortProposals(proposals: IProposal[], filter: ProposalFilter): I
     return proposals?.sort((proposal1, proposal2) => orderFunction(proposal1, proposal2, isAscending)) ?? []
 }
 
+function sortByPhaseAndMilestone(proposal1: IProposal, proposal2: IProposal, asc: boolean): number {
+    return sortByPhase(proposal1, proposal2, asc) || sortByMilestone(proposal1, proposal2, asc)
```

Comment:
as far as i can see, the second part of the condition will never be called, because  `sortByPhase` will either return 1 or -1. It should return 0 if the phase is the same

Reply:
Exactly, so if it is `0` it evaluates to false, which means it will get sorted on the secondary criteria which is the milestone.